### PR TITLE
make sure api works as expected

### PR DESCRIPTION
**Pull request by nshimiye**

### CODE REVIEW

File: src/main/java/me/marcellin/timechecker/api/v1/time/TimeController.java [hunk 1]
```diff
@@ -16,13 +19,14 @@
     @Autowired
     private TimeService timeService;
 
-    @RequestMapping("/{queryCity}")
-    public TimeModel check(
+    @RequestMapping(value = "/{queryCity}", method = RequestMethod.GET,  produces = "application/json")
```
Comment (dgongal) [inline src/main/java/me/marcellin/timechecker/api/v1/time/TimeController.java:22]: why do you have queryCity?

Comment (nshimiye) [reply src/main/java/me/marcellin/timechecker/api/v1/time/TimeController.java:22]: spring uses that as a placeholder for data passed in through the url

example when you type `http://example.com/John Smith` in the browser, Spring will assign queryCity to John Smith

Comment (dgongal) [reply src/main/java/me/marcellin/timechecker/api/v1/time/TimeController.java:22]: Got it. Thanks.

Comment (nshimiye) [reply src/main/java/me/marcellin/timechecker/api/v1/time/TimeController.java:22]: forgot to add the link to where queryCity is being used
https://github.com/nshimiye/time-checker/pull/2/files/bf25ff85ff74980c6070e35d8e7e07beecb7e638#diff-7a5f9c3062a22b93b035c04266c85fbfR24

**Pull request merged**